src/main/java/com/checkmarx/flow/service/SastScanner.java

```diff
@@ -67,22 +76,38 @@ public ScanResults scan(ScanRequest scanRequest) {
             Integer projectId = cxScanParams.getProjectId();
 
             log.info("Checking if there is any existing scan for Project: {}", projectId);
-            Integer existingScanId = cxService.getScanIdOfExistingScanIfExists(projectId);
+                Integer existingScanId = cxService.getScanIdOfExistingScanIfExists(projectId);
+
+            String scanComment = getComment(scanRequest);
```

Comment:
getComment => helperService.getCxComment get input from external script and should override other comments if exist.
so I think it should be instead of the hardcoded detailed comment and not in addition

Reply:
Hi Aviv,

I don't think I wrote anything related to 

cxService.getScanIdOfExistingScanIfExists(projectId)

I'm going to take a look at see why its flagging that.

I did override the getCxComment() line and matters. We're storing the scan information in the scan comment now but that only happens when post-back mode is turned on.